fabric-biomes-v1/src/main/resources/fabric-biomes-v1.mixins.json

```diff
@@ -3,17 +3,16 @@
   "package": "net.fabricmc.fabric.mixin.biome",
   "compatibilityLevel": "JAVA_8",
   "mixins": [
-    "DimensionOptionsAccessor",
-    "DimensionTypeAccessor",
     "MixinAddEdgeBiomesLayer",
     "MixinAddHillsLayer",
     "MixinAddRiversLayer",
-    "MixinBiomeSource",
     "MixinSetBaseBiomesLayer",
-    "MixinMinecraftDedicatedServer",
     "MixinMultiNoiseBiomeSource",
-    "MixinVanillaLayeredBiomeSource",
-    "MultiNoiseBiomeSourceAccessor"
+    "VanillaLayeredBiomeSourceAccessor",
+    "MultiNoiseBiomeSourceAccessor",
+    "BiomeSourceAccessor",
+    "BiomesAccessor",
+    "AddHillsLayerAccessor"
```

Comment:
please put these in alphabetical order. This reverse ordering adds a bunch of noise to the pull request

Reply:
Done (also: it sucks that the suffix/prefix PR is still in limbo, this is mixed between the two and needs to be changed with that PR)